mix/CodeModel.h

```diff
@@ -131,6 +132,8 @@ class CodeModel: public QObject
 	bool hasContract() const;
 	/// Apply text document formatting. @todo Move this to editor module
 	void updateFormatting(QTextDocument* _document);
+	/// Get contract code by url. Contract is compiled on first access and cached
+	dev::bytes const& getStdContractCode(QString const& _url);
```

Comment:
It also adds the contract to a cache of standard contracts. So, not that generic